### PR TITLE
refactor(glyphs): move information out of mocks, improve type sense, …

### LINKED ISSUE #5443: refactor glyphs to be simpler. remove from vuex state

### PR DESCRIPTION
**Pull request by josephmcg**
…comp api upgrade

### What this PR does 📖
- title
- fix bug where ephemeral message wasn't getting removed after send
- only partially done. wanted to break it up across PRs
- temporarily remove 'recently used' glyphs. This was being stored in vuex store and persisted in local storage. that approach will not support multiple devices or platforms.

### Which issue(s) this PR fixes 🔨
- Resolve #5443 

### Special notes for reviewers 🗒️
- 

### Additional comments 🎤
-

### CODE REVIEW

File: components/views/chat/enhancers/glyphs/Glyphs.vue [hunk 1]
```diff
@@ -1,31 +1,40 @@
-<template src="./Glyphs.html"></template>
-<script lang="ts">
-import Vue from 'vue'
-import VirtualList from 'vue-virtual-scroll-list'
```
Comment (molimauro) [inline components/views/chat/enhancers/glyphs/Glyphs.vue:4]: Can we remove this lib if not needed anymore?

**Review by molimauro — APPROVED** [review]
Approved but we have problems with the preview, for some reasons we cannot create accounts, not related to this though.

**Review by phillsatellite — APPROVED** [review]

**Comment by luisecm** [comment]
PR should be good to go on the cypress side, since the cypress GH actions failing now are related to iridium things that we are fixing now

File: components/views/chat/enhancers/glyphs/Glyphs.vue (hunk 1 above)
Comment (josephmcg) [reply components/views/chat/enhancers/glyphs/Glyphs.vue:4]: yes! thank you

**Pull request merged**